src/mono/mono/metadata/debug-mono-ppdb.h

```diff
@@ -74,4 +74,7 @@ mono_ppdb_is_embedded (MonoPPDBFile *ppdb);
 MONO_COMPONENT_API MonoPPDBFile*
 mono_create_ppdb_file (MonoImage *ppdb_image, gboolean is_embedded_ppdb);
 
+MONO_COMPONENT_API gboolean
+get_pe_debug_info_full (MonoImage *image, guint8 *out_guid, gint32 *out_age, gint32 *out_timestamp, guint8 **ppdb_data,
```

Comment:
does this need to be MONO_COMPONENT_API ?

Reply:
Thanks! Addressed in this PR: https://github.com/dotnet/runtime/pull/82587/